ffi-cdecl/include/mxcfb-pb.h

```diff
@@ -107,20 +107,22 @@ struct mxcfb_rect {
 #define EPDC_WFTYPE_GC16			2
 #define EPDC_WFTYPE_GC4				3
 #define EPDC_WFTYPE_A2				4
-#define EPDC_WFTYPE_GL16			5
+#define EPDC_WFTYPE_GL16			5 // On B288 this is very fast, but flickers, interferes with HWROT.
 #define EPDC_WFTYPE_A2IN			6
 #define EPDC_WFTYPE_A2OUT			7
-#define EPDC_WFTYPE_DU4				8
-#define EPDC_WFTYPE_AA				9
-#define EPDC_WFTYPE_AAD				10
-#define EPDC_WFTYPE_GC16HQ			15
+#define EPDC_WFTYPE_DU4				8 // not B288
+#define EPDC_WFTYPE_AA				9 // not B288
+#define EPDC_WFTYPE_AAD				10 // not B288
+#define EPDC_WFTYPE_GS16			14 // B288 only. More aggressive than GC16, less than GL16.
```

Comment:
Just for information, is that one constant buried somewhere in the B288 kernel?

Reply:
yep, https://github.com/pocketbook/kernel-b288/blob/master/drivers/video/sunxi/disp2/disp/eink.h